main.py

```diff
@@ -724,6 +766,12 @@ def update_outs_parallel(
         default=None,
         help="(finetuning only) Per-device and per-forward-pass batch size used to accumulate global --batch_size",
     )
+    parser.add_argument(
```

Comment:
Do we have a pair of applex-to-apples evaluations where validation early stopping improves final model, as opposed to naive (previous) early stopping? If yes, please attach links to wandb experiments

Reply:
There are no apples-to-apples comparison. 
I observed that the training in generally more robust with this option to the choice of learning rate.
The improvement is typically of order ~0.02 - 0.05 compared to our best runs.
In my opinion, this option is better than `relative_mse_tolerance`.

Reply:
This run 
https://wandb.ai/rock-and-roll/PPL_LLAMA_2/runs/anhk6jv6?nw=nwuserspiridon_sun_rotator - wikitext2 ppl = 6.22
is better than what we had in the paper
https://wandb.ai/rock-and-roll/PPL_LLAMA_2/runs/whmdskj8?nw=nwuserspiridon_sun_rotator - wikitext2 ppl = 6.31